src/core/variables.tf

```diff
@@ -1588,3 +1588,16 @@ variable "devops_agent_balance_zones" {
   default     = false
   description = "(Optional) True if the devops agent instances must be evenly balanced between the configured zones"
 }
+
+
+variable "enable_logos_backup" {
```

Comment:
This variable is used for all the storage accounts or only for the sftp?

Reply:
only for logos_donation_flows_sa